doc/translation.md

```diff
@@ -36,7 +36,7 @@ would become
 
 In order to for the new translations to appear in your copy of DRIVER, we suggest that you [make a pull request](https://github.com/WorldBank-Transport/DRIVER/pulls) with your changes. You may need to get a developer or system administrator to help you do this.
 
-Once your language is available in DRIVER, you will need to update your copy of DRIVER to use the new version and enable your language. Please see [System Administration](system-administration.md) for how to do this.
+Once your language is available in DRIVER, you will need to update your copy of DRIVER to use the new version and enable your language. Please see [System Administration](system-administration.md) for how to do this. In particular, you will need to update list of languages for your DRIVER instance using the [Configuration Wizard](https://github.com/WorldBank-Transport/DRIVER/blob/develop/doc/system-administration.md#configuration-wizard).
```

Comment:
We should use a relative path for this like we do with the other link. A relative path would mean a user who forks this repository or views this page on a branch other than `develop` won't be directed into a potentially different context that might not fit what they're looking for:
```suggestion
Once your language is available in DRIVER, you will need to update your copy of DRIVER to use the new version and enable your language. Please see [System Administration](system-administration.md) for how to do this. In particular, you will need to update list of languages for your DRIVER instance using the [Configuration Wizard](system-administration.md#configuration-wizard).
```